api-spec/WFS3-core.yaml

```diff
@@ -0,0 +1,478 @@
+    content:
+      type: object
+      required:
+        - links
+        - collections
+      properties:
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/link'
+          example:
+            - href: 'http://data.example.org/collections.json'
+              rel: self
+              type: application/json
+              title: this document
+            - href: 'http://data.example.org/collections.html'
+              rel: alternate
+              type: text/html
+              title: this document as HTML
+            - href: 'http://schemas.example.org/1.0/foobar.xsd'
+              rel: describedBy
+              type: application/xml
+              title: XML schema for Acme Corporation data
+        collections:
+          type: array
+          items:
+            $ref: '#/components/schemas/collectionInfo'
+    collectionInfo:
+      type: object
+      required:
+        - name
+        - links
+      properties:
+        name:
+          description: 'identifier of the collection used, for example, in URIs'
+          type: string
+          example: buildings
+        title:
+          description: human readable title of the collection
+          type: string
+          example: Buildings
+        description:
+          description: a description of the features in the collection
+          type: string
+          example: Buildings in the city of Bonn.
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/link'
+          example:
+            - href: 'http://data.example.org/collections/buildings/items'
+              rel: item
+              type: application/geo+json
+              title: Buildings
+            - href: 'http://example.org/concepts/building.html'
+              rel: describedBy
+              type: text/html
+              title: Feature catalogue for buildings
+        extent:
+          $ref: '#/components/schemas/bbox'
+        crs:
+          description: >-
+            the list of coordinate reference systems supported by the service;
+            the first item is the default coordinate reference system
+          type: array
+          items:
+            type: string
+          default:
+            - 'http://www.opengis.net/def/crs/OGC/1.3/CRS84'
+    bbox:
+      type: object
+      required:
+        - bbox
+      properties:
+        crs:
+          type: string
+          enum:
+            - 'http://www.opengis.net/def/crs/OGC/1.3/CRS84'
+          default: 'http://www.opengis.net/def/crs/OGC/1.3/CRS84'
+        bbox:
+          description: 'west, north, east, south edges of the bounding box'
+          type: array
+          minItems: 4
+          maxItems: 4
+          items:
+            type: number
+            minimum: -180
+            maximum: 180
+          example:
+            - -180
+            - -90
+            - 180
+            - 90
+    featureCollectionGeoJSON:
+      type: object

```

Comment:
I think a feature collection also needs a type property set to FeatureCollection. I think that is usually done with an enum in openapi?